run_snakemake.py

```diff
@@ -201,8 +201,11 @@ def default_vaxrank_targets(config):
         get_output_dir(config),
         "vaccine-peptide-report_%s_%s" % (mhc_predictor, vcfs))
     # removed 'pdf' from this list until we install wkhtmltopdf in Docker image
-    return ['%s.%s' % (path_without_ext, ext) for ext in ('txt', 'json')]
-
+    return [
```

Comment:
might be cleaner to add the annotated csv target as part of a separate function, or generally elsewhere in the file with a comment - it's not technically output as part of vaxrank